### PR TITLE
EdgeHosting Solution to Cloudbolt sending messages to rabbitMQ

### PR DESCRIPTION
**Pull request by blueharford**
Rick, here is the plugin i created for sending messages to rabbitmq

### CODE REVIEW

File: actions/cloudbolt_plugins/send_to_rabbitmq/send_to_rabbitmq.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import pika
+import sys
+
+from common.methods import set_progress
+from jobs.models import Job
+
+def run(job, logger=None, **kwargs):
+    status = job.status
+    connection = pika.BlockingConnection(pika.ConnectionParameters(
+            host='<rabbitmq ip>'))
```
Comment (rkilcoyne) [inline actions/cloudbolt_plugins/send_to_rabbitmq/send_to_rabbitmq.py]: Let's make this line to parameterize this plugin.
host='{{ rabbitmq_ip }}'))

File: actions/cloudbolt_plugins/send_to_rabbitmq/send_to_rabbitmq.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+	cpu = server.cpu_cnt
+        channel = connection.channel()
+        channel.queue_declare(queue='<queuename>', durable=True)
+        job.set_progress("Sending Message that %s resources changed..." % server.hostname)
+        channel.basic_publish(exchange='<exchangename>',
+                      routing_key='<routingkey>',
+                      body="Server:{}\nMem:{}\nCpu:{}\nStatus:{}".format(hostname, mem_size, cpu, status),
+                      properties=pika.BasicProperties(
+                         delivery_mode = 2, # make message persistent
+                      ))
```
Comment (rkilcoyne) [inline actions/cloudbolt_plugins/send_to_rabbitmq/send_to_rabbitmq.py]: lines 17, 19, and 20 should also be parameterized with variable names. I suggest:
{{ queue_name }}
{{ exchange_name }}
{{ routing_key }}

File: actions/cloudbolt_plugins/send_to_rabbitmq/README.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+## Requirements
+
+To run this code you need `pika` library version 0.10.0 To install it run
+
+    pip install pika==0.10.0
+
+You may first need to run
+
+    easy_install pip
```
Comment (rkilcoyne) [inline actions/cloudbolt_plugins/send_to_rabbitmq/README.md]: easy_install not needed -- go ahead and remove lines 9-10.

**Comment by rkilcoyne** [comment]
Just asked for a few little tweaks to parameterize your script. All and all -- GREAT PR!

**Comment by blueharford** [comment]
@rkilcoyne thanks! made the changes you requested.

**Comment by Lingnik** [comment]
Thanks, @blueharford ! Sorry for the delay in merging this.

**Pull request merged**